syntax = "proto3";

import "google/protobuf/any.proto";

message SessionEvent {
	google.protobuf.Any event = 1;
}

message EventPing {}

message EventServerHandshake {
    string version = 1;
    sint32 database_version = 2;
}

message EventDatabase {
    bytes database = 1;
}
